core/src/com/projectkorra/projectkorra/ability/WaterAbility.java

```diff
@@ -374,6 +399,28 @@ public static void playWaterbendingSound(final Location loc) {
 		}
 	}
 
+	public static void updateSourceBlock(final Block sourceBlock) {
+		updateSourceBlock(sourceBlock, (block) -> {
```

Comment:
Same here. I don't understand the point of making it into a consumer, just to call it

Reply:
- Addressed in commit https://github.com/ProjectKorra/ProjectKorra/pull/1329/commits/b1e3a4544ed49bff022046efb912511950558c7d.